src/did.ts

```diff
@@ -59,6 +59,11 @@ export interface VerifyJWSOptions {
    * Cacao OCAP to verify the JWS with.
    */
   capability?: Cacao
+
+  /**
+   * Number of seconds that a revoked key stays valid for after it was revoked
+   */
+  revocationPhaseOut?: number
```

Comment:
I have a strong belief that all time-based arguments should have their units included in the name.  Otherwise it's too easy to pass in milliseconds where seconds are expected, etc.  Can we rename this arg to `revocationPhaseOutSecs`?

Reply:
I am okay with that but we do have: https://linear.app/3boxlabs/issue/NET-1184/use-proper-unix-timestamp-format-in-js-did

The renaming can/should be done there

Reply:
a perfect example of how easy it is to use the wrong units when they're not included in the argument names!

I'll file some tickets